src/server/game/Spells/SpellInfo.cpp

```diff
@@ -2270,6 +2270,15 @@ void SpellInfo::_LoadSpellDiminishInfo()
                 // Screams of the Dead (King Ymiron)
                 else if (Id == 51750)
                     return DIMINISHING_NONE;
+                // Triggered trample aura (ToC 5)
+                else if (Id == 67868)
+                    return DIMINISHING_NONE;
+                // The Black Knight's Death's Respite (ToC 5)
+                else if (Id == 67745)
+                    return DIMINISHING_NONE;
+                // The Black Knight's Death's Respite (Heroic ToC 5)
+                else if (Id == 68306)
+                    return DIMINISHING_NONE;
```

Comment:
I am pretty certain there shouldn't be any diminishing return for spells casted by creatures.